packages/functional-tests/lib/fixtures/standard.ts

```diff
@@ -112,3 +117,18 @@ export async function newPages(browser: Browser, target: BaseTarget) {
   const page = await context.newPage();
   return createPages(page, target);
 }
+
+// When running tests that utilize Sync (sign-in/out), we need to run them in a
+// completely different browser, otherwise we will get timeout issues.
+// The main cause of this is that Sync sets a property
+// `identity.fxaccounts.lastSignedInUserHash` to the last
+// user signed in. On subsequent login to Sync, a dialog is prompted for the user
+// to confirm. Playwright does not have functionality to click browser ui.
+export async function newPagesForSync(target: BaseTarget) {
```

Comment:
Note that Playwright workers launch new browsers but the old `about:config` values are still set.